src/main/java/com/academia/library/mapper/AuthorMapper.java

```diff
@@ -0,0 +1,15 @@
+package com.academia.library.mapper;
+
+import com.academia.library.dto.AuthorDto;
+import com.academia.library.model.Author;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel="spring")
+public interface AuthorMapper {
+    AuthorMapper INSTANCE =  Mappers.getMapper(AuthorMapper.class);
```

Comment:
I recommended you avoid this approach. Please simplify your code
https://www.baeldung.com/mapstruct

Reply:
Ok, thank. 
1 question - how better create mapper - interface or abstract?